backend/menifest/prometheus-deployment.yaml

```
@@ -0,0 +1,63 @@
+---
+# Prometheus PersistentVolumeClaim (PVC)
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: prometheus-data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 2Gi  # ✅ Fixed PVC storage request
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)